gateway-service-definitions/src/main/resources/services/livy/0.4.0/rewrite.xml

```diff
@@ -49,4 +49,10 @@
     <rewrite template="{$frontend[url]}/livy/ui/"/>
   </rule>
 
+  <filter name="LIVYSERVER/livy/outbound/headers">
```

Comment:
Where is this filter getting called from ? we need a corresponding entry in service.xml
```xml
    <route path="/livy">
        <rewrite apply="LIVYSERVER/livy/outbound/headers" to="response.headers"/>
    </route>
```

Reply:
Sure Added the relevant change.
But not sure why it is working in my test cluster without calling this filter explicitly in route path,  are these filters/rules applied by default based on matched pattern ?

Reply:
They should not be, that is weird.